### DISCUSSION TITLE
lsp-proxy vs lsp_proxy

### DISCUSSION THREAD

**Discussion by kkkrist** [rome/tools#4487] [General]
v12.1.0 seems to have changed the `lsp-proxy` command to `lsp_proxy` (hyphen vs. underscore). Is this intentional? The docs still use the hyphen. In any case, it breaks some editor integrations (nvim-lspconfig). Do downstream projects need to adapt?

```shell
$ ~/bin/rome-12.1.0 lsp-proxy
flags/invalid ━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━

  ✖ Failed to parse CLI arguments.
    
    Caused by:
      No such command: `lsp-proxy`, did you mean `lsp_proxy`?
```

**Comment by ematipico** [comment]
This definitely a bug!
